### ISSUE TITLE
bert

### ISSUE THREAD

**Issue by whyseu** [linzzzzzz/Chinese-Text-Classification-Pytorch#7]
search_space = {
'learning_rate': tune.loguniform(1e-5, 3e-5),
'num_epochs': tune.randint(2, 5),
'pad_size': tune.randint(128, 512),
'batch_size': tune.randint(8, 32)
#'dropout': tune.uniform(0, 0.5),
#'hidden_size': tune.randint(32, 257),
#'num_layers': tune.randint(1,3)
}
Bert模型还有pad_size和batch_size参数，我可以这么自己增加参数吗？

**Comment by whyseu** [comment]
search_space = {
    'learning_rate': tune.loguniform(1e-5, 3e-5),
    'num_epochs': tune.randint(2, 5),
    'pad_size': tune.randint(128, 512),
    'batch_size': tune.randint(8, 32)
    #'dropout': tune.uniform(0, 0.5),
    #'hidden_size': tune.randint(32, 257),
    #'num_layers': tune.randint(1,3)
}
Bert模型还有pad_size和batch_size参数，我可以这么自己增加参数吗？

**Comment by whyseu** [comment]
![image](https://github.com/user-attachments/assets/3e0b9f72-ee7f-4512-933c-0e49cf5efc0a)
无论上面的参数，还是下面的参数配置，基本都是这种报错
search_space = {
'learning_rate': tune.loguniform(1e-5, 3e-5),
'num_epochs': tune.randint(2, 5)
}
